CodeGeneration/Sources/SyntaxSupport/Child.swift

```diff
@@ -58,16 +58,14 @@ public enum ChildKind {
 /// A child of a node, that may be declared optional or a token with a
 /// restricted subset of acceptable kinds or texts.
 public class Child {
+  /// If the child has been renamed, its old, now deprecated, name.
+  private let deprecatedName: String?
+
```

Comment:
Any reason why you changed the order of `deprecatedName` and `name`?

Reply:
Oh yeah :/ - habit from all my projects where we are trying to separate `public` properties from `private` ones. Should I revert the changes?

Reply:
Done